roxie/ccd/ccdprotocol.cpp

```diff
@@ -1739,9 +1739,12 @@ class RoxieSocketWorker : public ProtocolQueryWorker
         unsigned remainingHttpConnectionRequests = global->maxHttpConnectionRequests ? global->maxHttpConnectionRequests : 1;
         unsigned readWait = WAIT_FOREVER;
 
+        SpanTimeStamp spanStartTimeStamp(true);
```

Comment:
If this is moved, it should be to after readAnother - otherwise all requests on a persistent connection will have the same time.  It also means that subsequent requests on a persistent connection will include the time waiting for the request to come through.  I am not sure that is desirable, but including the read time is worthwhile.
@mckellyln thoughts?

Reply:
Agreed, was forced to move up to account for the pre-query tasks.
This type of scenario usually means there's a missing span relationship, but I think we want to avoid the extra overhead and include pre-query tasks as events of the request span.

Reply:
Yes, that was understood and acknowledged.

Do you want to discuss if moving the timing up is a good change, or are you asserting otherwise?